packages/auth/amplify_auth_cognito_test/test/state/configuration_state_machine_test.dart

```diff
@@ -82,5 +84,72 @@ void main() {
 
       await stateMachine.close();
     });
+
+    test('reads existing analytics metadata if analytics is configured',
+        () async {
+      const testEndpointId = 'testEndpointId';
+
+      // Add state machine dependencies.
+      stateMachine.addInstance(
+        EndpointInfoStoreManager(
+          store: MockSecureStorage()
+            ..write(
+              key: testPinpointAppId + EndpointStoreKey.endpointId.name,
+              value: testEndpointId,
+            ),
```

Comment:
Remove this and call init in here instead

Reply:
No, I mean the logic for where to store the endpoint should not be hardcoded here.